.github/workflows/tests-nightly.yml

```diff
@@ -164,27 +186,21 @@ jobs:
         env:
           version: ${{github.event.client_payload.version}}
         run: |
-              echo "::set-output name=ref::$(echo $version | grep -ohE "[0-9]\.[0-9]+" )" 
+              echo "::set-output name=ref::$(echo $version | grep -ohE "[0-9]\.[0-9]+" )"
+
       - name: Checkout
         uses: actions/checkout@v3
         with:
           ref: ${{ steps.branch.outputs.ref }}
-      
-      - name: Get formatted version
-        id: version
-        env:
-          version: ${{github.event.client_payload.version}}
-        run: |
-              echo "::set-output name=formatted_version::$(echo $version | grep -ohE "[0-9]\.[0-9]+" | sed s/\\.// )" 
               
       - name: Cypress run - automatically
         id: cypress
         continue-on-error: true
         uses: cypress-io/github-action@v4
         env:
-          API_URI: https://v${{ steps.version.outputs.formatted_version }}.staging.saleor.cloud/graphql/
+          API_URI: ${{ needs.get-domain.outputs.domain }}graphql/
           APP_MOUNT_URI: ${{ secrets.APP_MOUNT_URI }}
-          CYPRESS_baseUrl: https://v${{ steps.version.outputs.formatted_version }}.staging.saleor.cloud/dashboard/
+          CYPRESS_baseUrl: ${{ needs.get-domain.outputs.domain }}dashboard/
```

Comment:
It's not a domain but an URL, this includes the file `getDomainForReleaseTests.js`. It's misleading